src/main/java/com/theelixrlabs/healthcare/constants/MessageConstants.java

```diff
@@ -5,11 +5,11 @@
  * These keys are mapped to actual messages in the properties file.
  */
 public class MessageConstants {
-    public static final String AADHAAR_NUMBER_SHOULD_BE_MANDATORY = "doctor.aadhaarNumber.mandatory";
-    public static final String DEPARTMENT_SHOULD_BE_MANDATORY = "doctor.department.mandatory";
+    public static final String AADHAAR_NUMBER_SHOULD_BE_MANDATORY = "{doctor.aadhaarNumber.mandatory}";
+    public static final String DEPARTMENT_SHOULD_BE_MANDATORY = "{doctor.department.mandatory}";
     public static final String DOCTOR_UNAVAILABLE = "The doctor is unavailable for the given ID";
-    public static final String FIRST_NAME_SHOULD_BE_MANDATORY = "doctor.firstName.mandatory";
+    public static final String FIRST_NAME_SHOULD_BE_MANDATORY = "{doctor.firstName.mandatory}";
     public static final String INVALID_AADHAAR_NUMBER_FORMAT = "doctor.aadhaarNumber.invalidFormat";
     public static final String INVALID_UUID = "invalid uuid";
```

Comment:
Move this INVALID_UUID to messages.properties

Reply:
yes changed it into message.properties